get_count-plugin/tests/simple/script.tcl

```diff
@@ -0,0 +1,30 @@
+yosys plugin -i ../../get_count.so
```

Comment:
the path shouldn't require any `../..`

Reply:
Fixed.

Reply:
Oh, now I see. `plugin -i name.so` looks in the specified directory and falls back to Yosys's shared folder.
I don't think we want `install` to depend on `test`. The tests are run in CI after `install` which copies all .so to Yosys's shared folder thus `plugin -i get_count.so` will work in CI.  
With your Makefile conda_packages will run the test before installing in contrast to other plugins.
I don't think this is needed.
My suggestions:
1. Don't call test before install
2. Change `plugin -i ../../get_count.so` to `plugin -i get_count.so`. For completeness as both options will work in CI. However, what if your case requires another plugin? It won't be available before install.

Reply:
Ok. Now I understand. Fixed.